### PR TITLE
[TOOLS-131] Add: freeradius helm chart with validation

### PR DESCRIPTION
**Pull request by dunaev-dmitry**

### CODE REVIEW

File: .github/workflows/helm-validation.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          path: /tmp/k8s-validators
+          key: ${{ runner.os }}-k8s-validators
+      - name: Test freeradius helm chart
+        working-directory: freeradius/helm
+        run: |
+          echo "Kubeval test"
+          helm template -f values.yaml . | /tmp/k8s-validators/kubeval --ignore-missing-schemas
+
+          echo "Kube-score test"
+          helm template -f values.yaml . | /tmp/k8s-validators/kube-score score - || true
```
Comment (brennerm) [inline .github/workflows/helm-validation.yml]: Think we should remove the `|| true`. I just added it for the cloud controller chart cause we agreed on ignoring kube-score results for some time. Don't think we should do the same here.

Comment (dunaev-dmitry) [reply .github/workflows/helm-validation.yml]: I'm not sure if we really want to do that. Based on https://github.com/Telecominfraproject/wlan-test-bss/runs/2674705279?check_suite_focus=true#step:5:16 , probably testing would simply fail all the time as kube-score is quite strict and I don't agree with everything it considers critical error

Comment (brennerm) [reply .github/workflows/helm-validation.yml]: In this case we can remove it completely from the pipeline.

Comment (dunaev-dmitry) [reply .github/workflows/helm-validation.yml]: Deleted kube-score and added Helm lint instead

**Review by brennerm — APPROVED** [review]

**Pull request merged**